src/workspace-tree/bazel_workspace_folder_tree_item.ts

```diff
@@ -178,22 +180,30 @@ export class BazelWorkspaceFolderTreeItem implements IBazelTreeItem {
       "",
     );
 
-    // Now collect any targets in the directory also (this can fail since
-    // there might not be a BUILD files at this level (but down levels)).
-    const queryResult = await new BazelQuery(
-      getDefaultBazelExecutablePath(),
-      workspacePath,
-    ).queryTargets(`:all`, {
-      ignoresErrors: true,
-      sortByRuleName: true,
-    });
-    const targets = queryResult.target.map((target: blaze_query.ITarget) => {
-      return new BazelTargetTreeItem(
-        this.resources,
-        this.workspaceInfo,
-        target,
-      );
-    });
+    let targets: BazelTargetTreeItem[] = [];
+    // Only get root targets if the root package is present
+    const rootBuildFile =
+      existsSync(join(workspacePath, "BUILD")) ||
+      existsSync(join(workspacePath, "BUILD.bazel"));
+
+    if (rootBuildFile) {
+      // Replace ...:* with :all in queryExpression
+      const queryExpressionRoot = queryExpression.replace(/\.{3}:\*/g, ':all');
```

Comment:
I am not sure whether I am missing something. Or maybe this changeset got corrupted by recent refactorings on main. But as far as I understand the code, there is no variable `queryExpression` here. Maybe you need to rebase onto latest `master`?